File: app/src/main/java/ntutifm/game/google/net/ApiClass/Camera.java
package ntutifm.game.google.net.ApiClass;

import com.google.gson.annotations.SerializedName;

public class Camera {
    @SerializedName("ID")
    private String Id ="ID";
    @SerializedName("Type")
    private String Type ="Type";
    @SerializedName("Road")
    private String Road ="Road";
    @SerializedName("Introduction")
    private String Introduction ="Introduction";
    @SerializedName("Session")
    private String Session ="Session";
    @SerializedName("Direction")
    private String Direction ="Direction";
    @SerializedName("Limit")
    private String Limit ="Limit";
    @SerializedName("Latitude")
    private Double Latitude = 25.0;
    @SerializedName("Longitude")
    private Double Longitude = 123.0;
    @SerializedName("Distance")
    private Integer Distance = 10000;

    public Camera(){
    }
    public String getId() {
        return this.Id;
    }
    public String getType() {return this.Type;}
    public String getRoad() {
        return this.Road;
    }
    public String getSession() {
        return this.Session;
    }
    public String getDirection() {
        return this.Direction;
    }
    public String getLimit() {return this.Limit;}
    public Double getLatitude() {return this.Latitude;}
    public Double getLongitude() {return this.Longitude;}
    public String getIntroduction(){return this.Introduction;}
    public Integer getDistance(){return this.Distance;}

}

File: app/src/main/java/ntutifm/game/google/net/apiClass/Camera.kt
package ntutifm.game.google.net.apiClass

import com.google.gson.annotations.SerializedName

data class Camera(
    @SerializedName("ID") val id: String,
    @SerializedName("Type") val type: String,
    @SerializedName("Road") val road: String?,
    @SerializedName("Introduction") val introduction: String,
    @SerializedName("Session") val session: String,
    @SerializedName("Direction") val direction: String,
    @SerializedName("Limit") val limit: String,
    @SerializedName("Latitude") val latitude: Double,
    @SerializedName("Longitude") val longitude: Double,
    @SerializedName("Distance") val distance: Int
)